src/genie/libs/parser/iosxe/tests/ShowLicenseSummary/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,14 @@
+expected_output = {
+   'license_usage':{
+      'network-advantage':{
+         'entitlement':'(C9300-48 Network Advan...)',
+         'count':'1',
+         'status':'IN USE'
+      },
+      'dna-advantage':{
+         'entitlement':'(C9300-48 DNA Advantage)',
```

Comment:
Now that you've updated the regex to exclude the brackets, please update the tests as well to pass the automated testing process

![image](https://user-images.githubusercontent.com/33130026/128441016-40fea312-2cb7-4b02-b9b1-25954c23c3f1.png)